apps/api/src/auth/service/auth.service.ts

```diff
@@ -131,6 +131,24 @@ export class AuthService {
     }
   }
 
+  /* istanbul ignore next */
```

Comment:
What does this do? @risv1

Reply:
That's cool! We are keeping that because we want to exclude the test coverage of these functions from the test coverage report

Reply:
> That's cool! We are keeping that because we want to exclude the test coverage of these functions from the test coverage report

Cool cool. got it.